cluster-install/install.sh

```diff
@@ -755,6 +755,7 @@ version_in_range() {
     if [ -z "$current" ]; then fatal "no current version given"; fi
     if [ -z "$min" ]; then fatal "no min version given"; fi
     if [ -z "$max" ]; then fatal "no max version given"; fi
+    if [ "${current%.0}" == "${max%.0}" ]; then return 0; fi
```

Comment:
This fixes an issue where k8s version `1.29.0` wasn't being accepted even if the max was set to `1.29`.